### PR TITLE
Swap rerun to use iframe-less Galaxy.display.

### PR DESCRIPTION
**Pull request by dannon**
The old urls and links still work, intentionally, for now.

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/jobs.py [hunk 1]
```diff
@@ -185,6 +186,16 @@ def outputs( self, trans, id, **kwd ):
         job = self.__get_job( trans, id )
         return self.__dictify_associations( trans, job.output_datasets, job.output_library_datasets )
 
+    @expose_api_anonymous
+    def build_for_rerun( self, trans, id, **kwd ):
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/jobs.py:190]: Can you add API documentation, e.g.:

```
"""
GET /api/jobs/{job_id}/build_for_rerun

What it does...
"""
```

Comment (dannon) [inline lib/galaxy/webapps/galaxy/api/jobs.py:190]: Good call, I knew I meant to go back and do something else after talking to @guerler about this :)

File: lib/galaxy/webapps/galaxy/api/jobs.py [hunk 2]
```diff
@@ -185,6 +186,29 @@ def outputs( self, trans, id, **kwd ):
...
+        """
+        * GET /api/jobs/{job_id}/build_for_rerun
+            returns a tool input/param template preopoulated with this job's
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/jobs.py]: s/preopoulated/prepopulated/

Comment (dannon) [inline lib/galaxy/webapps/galaxy/api/jobs.py]: I saw it.  Working on the test cases.

File: lib/galaxy/webapps/galaxy/api/jobs.py [hunk 3]
```diff
@@ -185,6 +186,29 @@ def outputs( self, trans, id, **kwd ):
         job = self.__get_job( trans, id )
         return self.__dictify_associations( trans, job.output_datasets, job.output_library_datasets )
 
+    @expose_api_anonymous
+    def build_for_rerun( self, trans, id, **kwd ):
+        """
+        * GET /api/jobs/{job_id}/build_for_rerun
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/jobs.py:192]: I just noticed that we both used `{job_id}` here while the function parameter and the doc below use `id`, should we standardize on the latter? All public methods in this file have the same problem.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/api/jobs.py:192]: Yep, noticed this when I was adding the route, just kept it the same as the rest of the file.  Another PR.

Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/jobs.py:192]: Sure, fine!

**Comment by nsoranzo** [comment]
:+1:

**Pull request merged**